### PR TITLE
Landing page gifs

### PR DESCRIPTION
**Pull request by DariaAzanovich**

### CODE REVIEW

File: workflow/landing-page/index.html [hunk 1]
```diff
@@ -312,10 +312,10 @@ <h3 >And here are some images for you...</h3>
                     <div class="cards-wrap">
                         <div class="card">
```
Comment (qwaseras) [inline workflow/landing-page/index.html:313]: Remove this cards from HTML and add them after loading data

File: workflow/landing-page/sources/sctript.js [hunk 2]
```diff
@@ -18,4 +18,79 @@ closeBtn.addEventListener('click', function () {
...
+
+let offset = 0,
+    currentOffset = cardsAmount;
+
+let searchArr = ['cute', 'cat'],
+    search = searchArr.join('+'),
+    regExpOffcet = new RegExp(/offset=\d*/);
+
+
+let url = new URL(`https://api.giphy.com/v1/gifs/search?q=${search}&api_key=${API_KEY}&limit=${cardsAmount}&offset=`);
```
Comment (qwaseras) [inline workflow/landing-page/sources/sctript.js:46]: move this variable definition into the uploadGifs method
or even create a separate method which accepts all needed params and returns an URL

File: workflow/landing-page/sources/sctript.js [hunk 3]
```diff
@@ -18,4 +18,79 @@ closeBtn.addEventListener('click', function () {
...
+
+uploadGifs(offset);
+
+async function uploadGifs(currOffset) {
+    url.search = url.search.replace(regExpOffcet, `offset=${currOffset}`);
+
+    await fetch(url)
```
Comment (qwaseras) [inline workflow/landing-page/sources/sctript.js:53]: dont use callback approach inside async func 
[Please read this](https://developer.mozilla.org/ru/docs/Learn/JavaScript/Asynchronous/Async_await#%D0%BF%D0%B5%D1%80%D0%B5%D0%BF%D0%B8%D1%81%D1%8B%D0%B2%D0%B0%D0%B5%D0%BC_promises_%D1%81_%D0%B8%D1%81%D0%BF%D0%BE%D0%BB%D1%8C%D0%B7%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D0%B5%D0%BC_asyncawait)

**Review by qwaseras — CHANGES_REQUESTED** [review]

File: workflow/landing-page/sources/sctript.js [hunk 4]
```diff
@@ -18,4 +18,104 @@ closeBtn.addEventListener('click', function () {
...
+
+let offset = 0,
+    currentOffset = offset;
+
+let search = ['cute', 'cat'].join('+');
+
+
+let imgBlackout = '<div class="img-blackout"><div class="link-btn"><a href="#" name="gif-link"><i class="fas fa-link"></i></a></div></div>';
+
+let cardText = '<p class="card-text">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Alias, deleniti, id quibusdam aut optio saepe soluta tempore neque voluptatum.</p>';
```
Comment (DariaAzanovich) [inline workflow/landing-page/sources/sctript.js:46]: Not sure about this elements (they're same for each card)

File: workflow/landing-page/sources/sctript.js [hunk 5]
```diff
@@ -18,4 +18,104 @@ closeBtn.addEventListener('click', function () {
...
+
+const cardsImg = document.getElementsByClassName('card-img');
+const cardsDate = document.getElementsByClassName('card-date');
+const cardContent = document.getElementsByClassName('card-content');
+const imgLink = document.getElementsByName('gif-link');
+
+const leftBtn = document.querySelector('.pag-left');
+const rightBtn = document.querySelector('.pag-right');
+
+const cardsAmount = 9;
```
Comment (DariaAzanovich) [inline workflow/landing-page/sources/sctript.js:36]: What about masonry layout? Using flexbox it's needed to set max-height. If change cardsAmount all'll break

File: workflow/landing-page/sources/sctript.js (hunk 4 above)
Comment (qwaseras) [reply workflow/landing-page/sources/sctript.js]: It's ok, let it be

**Review by qwaseras — APPROVED** [review]

**Pull request merged**